e2e/video-modal.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+const currentUrlPath =
+  '/learn/javascript-algorithms-and-data-structures/basic-javascript/increment-a-number-with-javascript';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(currentUrlPath);
+  await page.getByTestId('get-help-dropdown').click();
+  await page.getByTestId('watch-a-video').click();
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
+test.describe('Exit Video Modal E2E Test Suite', () => {
+  test('Verifies the Correct Rendering of the Video Modal', async ({
+    page
+  }) => {
+    await expect(page.getByTestId('video-modal-title')).toBeVisible();
+    await expect(
+      page.getByTestId('video-modal-video-player-iframe')
+    ).toBeVisible();
+    await expect(
+      page.getByText(translations.learn['scrimba-tip'])
+    ).toBeVisible();
+    await expect(
+      page.getByRole('button', { name: translations.buttons.close })
+    ).toBeVisible();
+    await expect(page).toHaveURL(currentUrlPath);
```

Comment:
It checks if we are still on the same page, I thought it would be useful. It will fail in case of redirect to another site or different page. Should  I remove it?

Reply:
Thank you @huyenltnguyen. If it is not possible according requirements I'll remove it